tests/acceptance/bootstrap/SpacesContext.php

```diff
@@ -254,21 +254,65 @@ public function getSpaceIdByName(string $user, string $spaceName): string {
 		return $space["id"];
 	}
 
+	/**
+	 * Retrieves the remoteItem ID for a given share in the Shares space.
+	 *
+	 * @param string $user
```

Comment:
method name is clear
```suggestion
	 * @param string $user
```